app/components/npq_separation/admin/outcomes_table_component.rb

```diff
@@ -19,7 +19,8 @@ def head
           "Completion date",
           "Submitted by provider",
           "Sent to TRA API",
-          "Recorded by TRA API",
+          "Recorded by API",
+          "",
```

Comment:
there's some `caption_text` logic in ECF, not sure if we want to bring across, don't think it's a must but just incase that was missed